File: 1-2/1-4.c
#include <stdio.h>

main() {
	float fahr, celsius;
	int lower, upper, step;

	lower = 0;
	upper = 300;
	step = 20;

	celsius = lower;
	printf("Celsius %18s", "Fahrenheit\n");
	while (celsius <= upper) {
		fahr = celsius * (9.0/5.0) + 32;
		printf("%3.0f %19.1f\n", celsius, fahr);
		celsius = celsius + step;
	}
}

File: 1-2/1-4.rs
fn main() {
	let (lower, upper, step) = (0.0, 300.0, 20.0);

	let mut celsius = lower;
	println!("Celsius {:>18}", "Fahrenheit");
	while celsius <= upper {
		let fahr = format!("{:.*}", 1, (celsius * (9.0/5.0) + 32.0));
		println!("{:>3} {:>19}", celsius, fahr);
		celsius += step;
	}
}
